packages/turbo-vsc/src/extension.ts

```diff
@@ -311,14 +307,14 @@ function updateJSONDecorations(editor?: TextEditor) {
     return;
   }
 
-  let isPipelineKey = false;
-  let pipelineDepth = -1; // indicates we're not in a pipeline block
+  let depth = 0; // indicates we're not in a pipeline block
+  let inPipeline = false; // we do not use this right now but could highlight tasks
 
-  const ranges: Range[] = [];
   visit(editor.document.getText(), {
-    onObjectProperty: (property, offset, length) => {
-      if (property === "pipeline") {
-        isPipelineKey = true;
+    onObjectProperty: (property, offset) => {
```

Comment:
can this not be the shorthand syntax?

```suggestion
    onObjectProperty(property, offset) {
```